generator/internal/scaffold_generator.cc

```diff
@@ -884,7 +884,7 @@ int main(int argc, char* argv[]) try {
       $library$::MakeConnection(/* EDIT HERE */));
 
   auto const project = google::cloud::Project(argv[1]);
-  for (auto r : client.List/*EDIT HERE*/(project.FullName()) {
+  for (auto r : client.List/*EDIT HERE*/(project.FullName())) {
```

Comment:
Thanks (he says after having been bitten by that, and each time blaming himself for an overzealous edit).